addon/instance-initializers/in-context.js

```diff
@@ -1 +1,53 @@
-export { default, initialize } from 'ember-cli-crowdin/instance-initializers/in-context';
```

Comment:
See https://github.com/outdoorsy/ember-cli-crowdin/pull/16 for why Cam had this in `/app`

Reply:
ohh.. interesting.. 🤔 there should be a better way for this tho, I'll look into it, thanks @danwenzel 🙌

Reply:
Done! 👍 thanks